### PR TITLE
Add docs on '?layout=embedded'

### LINKED ISSUE #89: Add documentation for Layout=embedded
https://github.com/RocketChat/Rocket.Chat/pull/4319

### PR DESCRIPTION
**Pull request by MartinSchoeler**
closes #89

### CODE REVIEW

File: _data/toc.yml [hunk 1]
```diff
@@ -154,6 +154,7 @@
       - Authentication
       - Commands
       - Events
+    - Layout Embedded
```
Comment (sampaiodiego) [inline _data/toc.yml]: Should this be changed to `Embedded Layout` as of [page's title](https://github.com/RocketChat/docs/pull/681/files#diff-21a9468c9abb601c7f3650003dd3ae19R1)

File: contributing/documentation/documentation-map/README.md [hunk 2]
```diff
@@ -193,6 +193,7 @@ Here you can also find what articles are incomplete and missing.
         - <span class="missing">[Authentication](../missing-and-outdated-list/index.html#Authentication)</span>
         - Commands
         - Events
+    - Layout Embedded
```
Comment (sampaiodiego) [inline contributing/documentation/documentation-map/README.md]: Should this be changed to `Embedded Layout` as of [page's title](https://github.com/RocketChat/docs/pull/681/files#diff-21a9468c9abb601c7f3650003dd3ae19R1)

File: developer-guides/layout-embedded/README.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Embedded Layout
+
+If you are embedding Rocket.Chat in your site, you can change Rocket.Chat's layout by adding `?layout=embedded` after your server url.
```
Comment (sampaiodiego) [inline developer-guides/layout-embedded/README.md]: please change all `url` to capital `URL` along whole page.

**Review by sampaiodiego — CHANGES_REQUESTED** [review]

**Comment by MartinSchoeler** [comment]
@sampaiodiego Fixed the reviews

**Review by sampaiodiego — APPROVED** [review]

**Pull request merged**